mitosheet/src/components/LoadingIndicator.tsx

```diff
@@ -1,43 +1,275 @@
+        }
+        case StepType.ReorderColumn: {
+            return 'Reording a column'
+        }
+        case StepType.FillNa: {
+            return 'Filling NaN values'
+        }
+        case StepType.FilterColumn: {
+            return 'Filtering a column'
+        }
+        case StepType.SetColumnFormula: {
+            return 'Setting a formula'
+        }
+        case StepType.DataframeDelete: {
+            return 'Deleting a dataframe'
+        }
+        case StepType.DataframeDuplicate: {
+            return 'Duplicating a dataframe'
+        }
+        case StepType.DataframeRename: {
+            return 'Renaming a dataframe'
+        }
+        case StepType.SimpleImport: {
+            return 'Importing a CSV'
+        }
+        case StepType.Sort: {
+            return 'Sorting a dataframe'
+        }
+        case StepType.Pivot: {
+            return 'Pivoting a dataframe'
+        }
+        case StepType.Merge: {
+            return 'Merging dataframes'
+        }
+        case StepType.Concat: {
+            return 'Concatenating dataframes'
+        }
+        case StepType.DropDuplicates: {
+            return 'Dropping duplicates'
+        }
+        case StepType.ChangeColumnDtype: {
+            return 'Changing a dtype'
+        }
+        case StepType.ChangeColumnFormat: {
+            return 'Formatting'
+        }
+        case StepType.SetCellValue: {
+            return 'Setting a cell value'
+        }
+        case StepType.BulkOldRename: {
+            return 'Bulk old rename'
+        }
+        case StepType.ExcelImport: {
+            return 'Import an Excel file'
+        }
+        case StepType.Graph: {
+            return 'Creating graph'
+        }
+        case StepType.GraphDuplicate: {
+            return 'Duplicating graph'
+        }
+        case StepType.GraphDelete: {
+            return 'Deleting graph'
+        }
+        case StepType.GraphRename: {
+            return 'Renaming graph'
+        }
+        case StepType.DeleteRow: {
+            return 'Deleting row'
+        }
+        case StepType.PromoteRowToHeader: {
+            return 'Promoting row to header'
+        }
+        case StepType.SplitTextToColumns: {
+            return 'Spliting column'
+        }
+        case StepType.Transpose: {
+            return 'Transposing dataframe'
+        }
+        case StepType.Melt: {
+            return 'Melting dataframe'
+        }
+        case UpdateType.Undo: {
+            return 'Undo previous edit'
+        }
+        case UpdateType.Redo: {
+            return 'Redo previous edit'
+        }
+        case UpdateType.Clear: {
+            return 'Clear all edits'
+        }
+        case UpdateType.ArgsUpdate: {
+            return 'Updating user profile';
+        }
+        case UpdateType.SaveAnalysisUpdate: {
+            return 'Saving current analysis'
+        }
+        case UpdateType.ReplayAnalysisUpdate: {
+            return 'Replaying previous analysis'
```

Comment:
This text is cut off. A bunch of these are probably cut off too: checking out previous step, promoting row to header, etc. 

<img width="529" alt="Screen Shot 2022-06-20 at 11 21 27 PM" src="https://user-images.githubusercontent.com/18709905/174709507-1cb55f3a-82e6-41c3-b43c-47b6da1672a5.png">